crypto/pemutil/pem_test.go

```diff
@@ -230,7 +230,7 @@ func TestReadCertificate(t *testing.T) {
 		{"testdata/notexists.crt", errors.New("open testdata/notexists.crt failed: no such file or directory")},
 		{"testdata/badca.crt", errors.New("error parsing testdata/badca.crt")},
 		{"testdata/badpem.crt", errors.New("error decoding testdata/badpem.crt: not a valid PEM encoded block")},
-		{"testdata/badder.crt", errors.New("error parsing testdata/badder.crt: asn1: syntax error: data truncated")},
+		{"testdata/badder.crt", errors.New("error parsing testdata/badder.crt: x509:")},
```

Comment:
Strip x509 so it also works on 1.15+

```suggestion
		{"testdata/badder.crt", errors.New("error parsing testdata/badder.crt:")},
```